### PR TITLE
Update CodeQL, update docs

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Update CodeQL - currently used version [are deprecated](https://github.com/kubeshop/botkube/actions/runs/2655370882). I used the default template provided by GitHub for defining CodeQL pipeline. I updated only the schedule to run at 09:00 UTC on Wed and removed unnecessary comments.
- Update docs - related PR https://github.com/kubeshop/botkube-docs/pull/95
 
## Testing

The branch was pushed to upstream, as a result the PR build uses the newest definition for CodeQL pipeline.
https://github.com/kubeshop/botkube/runs/7299143741?check_suite_focus=true

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ We gratefully welcome improvements to [documentation](https://botkube.io/ "Go to
 
 ## Contributing to documentation
 
-You can contribute to documentation by following [these instructions](https://github.com/kubeshop/botkube-docs#contributing "Contributing to BotKube Docs")
+Follow the [botkube-docs/CONTRIBUTING.md](https://github.com/kubeshop/botkube-docs/blob/main/CONTRIBUTING.md) to learn how to contribute to documentation.
```
Comment (pkosiec) [inline CONTRIBUTING.md:11]:
```suggestion
Follow the [botkube-docs/CONTRIBUTING.md](https://github.com/kubeshop/botkube-docs/blob/main/CONTRIBUTING.md) file to learn how to contribute to documentation.
```

File: .github/workflows/codeql.yaml [hunk 2]
```diff
@@ -2,22 +2,46 @@ name: Code Scanning
 
 on:
   push:
+    branches: [ "main" ]
   pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
```
Comment (pkosiec) [inline .github/workflows/codeql.yaml:8]: IMHO we should narrow down the types of events (see the full list here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request=)

```suggestion
    branches:
      -  main
    types: [ opened, synchronize, reopened ]
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ We gratefully welcome improvements to [documentation](https://botkube.io/ "Go to
```
Comment (pkosiec) [inline CONTRIBUTING.md:8]: Please update also line 113 - replace `http://` with `https://` 🙂

**Review by pkosiec — APPROVED** [review]
LGTM

File: .github/workflows/codeql.yaml (hunk 2 above)
Comment (mszostok) [reply .github/workflows/codeql.yaml:8]: Yep, but what I checked by default it uses only `opened, synchronize, reopened`. I also tested that by adding the label. Also your comments didn't trigger the pipeline. IMO it's better to stay with the defaults in this case.

**Pull request merged**